source/system-administrators/activities/autoclustering-with-studio.rst

```diff
@@ -216,26 +216,35 @@ You can also verify from Studio that there are two nodes in the cluster by click
 Setup the Studio Arbiter
 ------------------------
 
-Whenever the number of Studios in the cluster is even numbered, the Studio Arbiter needs to be started.  To setup the Studio Arbiter:
+Whenever the number of Studios in the cluster is an even numbered, the Studio Arbiter needs to be started.  
```

Comment:
Typo? 
Whenever the number of Studios in the cluster is an even `number`, the Studio Arbiter needs to be started.